indigo_social/templates/indigo_social/_achievement.html

```diff
@@ -2,10 +2,10 @@
 
 <div class="achievement"
   {% if award %}
-    data-toggle="tooltip" data-placement="right" data-html="true"
+    data-bs-toggle="tooltip" data-placement="right" data-html="true"
     title="<p>Awarded {{ award.awarded_at|timesince }} ago </p> <p>{{ award.description }} </p>"
   {% endif %}>
-  <div class="achievement-icon badge-{{ badge.nature }} badge-{{ badge.slug }} {{ badge.css_class }}" style="background-image: url({% static badge.image %});"></div>
+  <div class="achievement-icon text-bg-{{ badge.nature }} text-bg-{{ badge.slug }} {{ badge.css_class }}" style="background-image: url({% static badge.image %});"></div>
```

Comment:
these must be retained because they relate to our internal badges, not bootstrap badges.